ote/ote/tasks/mmdetection/parameters.py

```diff
@@ -0,0 +1,65 @@
+import os
+
+from ote.interfaces.parameters import BaseTaskParameters
+
+
+def is_field(name, t):
+    if hasattr(t, name):
+        return True
+    return False
+
+
+def build_train_parameters(train_kwargs_dict, configs_root):
```

Comment:
Please use `ote.modules.arg_converters.convert_args_to_parameters` here in the next PR